temp/test_vtk.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import pickle
+import os
+from gns.train import rendering
+from omegaconf import DictConfig
+import pyvista as pv
+
+#configuration dictionary for VTK rendering
+def cfg_vtk():
+    return DictConfig({
+        'rendering': {
+            'mode': 'vtk'
+    }
+    })
+
+#function to count the no of files with a specific extension
+#in a directory
+def n_files(dir, extension):
+    files = os.listdir(dir)
+    each_file = []
+
+    for file in files:
+        if file.endswith(extension):
+            each_file.append(file)
+
+    return len(each_file)
+
+#define parameters for simulation
+n_trajectories = 1
+n_timesteps = 2
+n_particles = 3
+dim = 2
+n_init_pos = 2
+
+# Generate random predictions and ground truth positions
+predictions = np.random.rand(n_timesteps, n_particles, dim)
+ground_truth_positions = np.random.randn(n_timesteps, n_particles, dim)
+loss = (predictions - ground_truth_positions)**2
+
+# rollout dictionary to store all relevant information
+rollout = {
+    "initial_positions": np.random.rand(n_init_pos, n_particles, dim),
+    "predicted_rollout": predictions, 
+    "ground_truth_rollout": ground_truth_positions,
+    "particle_types": np.full(n_particles, 5), 
+}
+
+# Metadata for the simulation
+metadata = {
+        "bounds": [[0.0, 1.0], [0.0, 1.0]]
+}
+
+# Define output directory and file name
+output_dir = './temp'
+file_name = 'temp.pkl'
```

Comment:
you previously used tempfile. Continue using that and deleting and not manually do these.